packages/core/src/new-api/internal/execution/transaction-types.ts

```diff
@@ -0,0 +1,87 @@
+/**
```

Comment:
This file was replaced by `packages/core/src/new-api/internal/new-execution/types/jsonrpc.ts`